app/src/main/cpp/src/components/transport_manager/include/transport_manager/bluetooth_le/ble_control_protocol.h

```diff
@@ -0,0 +1,34 @@
+#ifndef SRC_COMPONENTS_TRANSPORT_MANAGER_INCLUDE_TRANSPORT_MANAGER_BLE_CONTROL_PROTOCOL_H_
+#define SRC_COMPONENTS_TRANSPORT_MANAGER_INCLUDE_TRANSPORT_MANAGER_BLE_CONTROL_PROTOCOL_H_
+
+#include <vector>
+#include <stdint.h>
+#include <string>
+
+namespace transport_manager {
+namespace transport_adapter {
+
+    enum class BleProtolActions {
+        INVALID_ENUM = 0,
+        ON_DEVICE_CONNECTED,
+        ON_DEVICE_DISCONNECTED,
+    };
+
+    using MsgData = std::vector<uint8_t>;
+
+    struct BleControlProtocol {
+
+        static BleProtolActions GetMessageActionType(const MsgData& message);
+        static std::string GetAddress(const MsgData& message);
+        static std::string GetName(const MsgData& message);
+
+    private:
+        static BleProtolActions GetActionFromString(const std::string& str);
+        static std::string GetActionString(const BleProtolActions action);
+    };
+
+}
+}
+
+
+#endif  // SRC_COMPONENTS_TRANSPORT_MANAGER_INCLUDE_TRANSPORT_MANAGER_BLE_CONTROL_PROTOCOL_H_
```

Comment:
@VadymLuchko please add an empty line

Reply:
fixed in 22caff304f02304d55c097802b850d8bca25a370